backend/app/schemas/position.py

```
@@ -1,27 +1,360 @@
+    today_change: Optional[Decimal] = Field(None, description="Today's price change in EUR")
+    today_change_percent: Optional[float] = Field(None, description="Today's percentage change")
+    last_calculated_at: datetime = Field(..., description="When position was last calculated")
+    exchange: Optional[str] = Field(None, description="Crypto exchange (for crypto assets)")
+    wallet_address: Optional[str] = Field(None, description="Wallet address (for crypto assets)")
+
+    @field_validator('ticker')
+    @classmethod
+    def validate_ticker(cls, v):
+        """Validate ticker format for both traditional assets and crypto."""
+        if not v:
+            raise ValueError("Ticker is required")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)